backend/GymBeam/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using GymBeam.Requests;
+using GymBeam.Response;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GymBeam.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthenticationController : ControllerBase
+    {
+        [HttpPost("Register")]
+        public ActionResult<UserResponse> Register([FromBody] UserRequest dto)
+        {
+            return new UserResponse
+            {
+                Id = 34,
+                Login = dto.Login,
+                DisplayName = dto.DisplayName,
+                Role = dto.Role
+            };
+        }
+
+        [HttpPost("Login")]
+        public ActionResult<UserResponse> Login([FromBody] UserLoginRequest dto)
+        {
+            return new UserResponse
+            {
+                Id = 25,
+                Login = dto.Login,
+                DisplayName = "testDisplayName",
+                Role = "User"
+            };
+        }
+
+        [HttpPost("{user_id:int}/Logout")]
```

Comment:
This is wrong because you suggest that id is for Authentication but this is user id
Authentication doesn't have ID property. Authentication is not entity and doesn't have any property